image/bin/build-parts

```diff
@@ -7,12 +7,13 @@ debug "parts: %s" "$*"
 # Don't cleanup apt runs inter-parts
 export cleanup=no
 
+declare rv
+
 parts=($(IFS=$'\n' run-parts --test "$@"))
 for part in "${parts[@]}"; do
 	e "[%s] exec" "$part"
 
-	if ! prefixout -dc -- "$part"; then
-    	rv=$?
+	if ! save-retval rv prefixout -dc -- "$part"; then
```

Comment:
Bugfix for logic error here.
If a step were to fail, which insofar had not happened, it would only display the notice that it had failed, but would exit with a successful retval due to `$?` not being set from the last conditional. Fun stuff, bash.